### ISSUE TITLE
Post: Show images from Pedernales Falls hike

### ISSUE THREAD

**Issue by meddlin** [meddlin/rushinglabs-blog#100]
Use the pictures and video from this hike to create some varied uses around the `<CenteredImage />` component.

- image left
- image right
- how to handle video outside of just embedding a YouTube video?
- component for "web" video?

**Comment by meddlin** [comment]
Putting a click handler on an image within React: [https://bobbyhadz.com/blog/react-onclick-on-image](https://bobbyhadz.com/blog/react-onclick-on-image)

```js
const App = () => {
  const handleClick = event => {
    console.log(event.target);
    console.log('Image clicked');
  };

  return (
    <div>
      <img src={MyImage} alt="horse" onClick={handleClick} />
    </div>
  );
};
```

**Comment by meddlin** [comment]
Making images bigger when clicked. This could likely be done while loading the image on a modal. So, (a) show modal, (b) make modal fill the screen, (c) image fills the modal

[https://www.geeksforgeeks.org/how-to-make-the-images-bigger-when-clicked/](https://www.geeksforgeeks.org/how-to-make-the-images-bigger-when-clicked/)

Throw these functions in click handlers

```js
// Get the img object using its Id
      img = document.getElementById("img1");
      // Function to increase image size
      function enlargeImg() {
        // Set image size to 1.5 times original
        img.style.transform = "scale(1.5)";
        // Animation effect
        img.style.transition = "transform 0.25s ease";
      }
      // Function to reset image size
      function resetImg() {
        // Set image size to original
        img.style.transform = "scale(1)";
        img.style.transition = "transform 0.25s ease";
      }
```

**Comment by meddlin** [comment]
Wait...does this modal idea work without doing anything else? Can we just get a modal that fills the screen??

> (a) show modal, (b) make modal fill the screen, (c) image fills the modal

**Comment by meddlin** [comment]
Solved by PR #105 

The modal question is put off for later for the sake of getting this post up.
